File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountactivate/SavingsAccountActivateMvpView.java
package com.mifos.mifosxdroid.online.savingsaccountactivate;

import com.mifos.api.GenericResponse;
import com.mifos.mifosxdroid.base.MvpView;

/**
 * Created by Tarun on 01/06/17.
 */
public interface SavingsAccountActivateMvpView extends MvpView {

    void showUserInterface();

    void showSavingAccountActivatedSuccessfully(GenericResponse genericResponse);

    void showError(String s);
}

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingsaccountactivate/SavingsAccountActivateMvpView.kt
package com.mifos.mifosxdroid.online.savingsaccountactivate

import com.mifos.api.GenericResponse
import com.mifos.mifosxdroid.base.MvpView

/**
 * Created by Tarun on 01/06/17.
 */
interface SavingsAccountActivateMvpView : MvpView {
    fun showUserInterface()
    fun showSavingAccountActivatedSuccessfully(genericResponse: GenericResponse?)
    fun showError(s: String?)
}